docs/api.md

```diff
@@ -0,0 +1,49 @@
+# Renovate API
+
+Renovate CE/EE exposes a REST API that can be used to interact programmatically with Renovate.
+
+## Enabling and Authentication
+
+The API can be enabled by setting the `MEND_RNV_ADMIN_API_ENABLED` environment variable to `true`.
+You must also configure an API secret by setting the `MEND_RNV_SERVER_API_SECRET` variable.
+
+Authentication is done via HTTP Auth, using the API secret as the password.
+For example if the secret is `renovateapi` then you would authenticate with:
+
+```
+  Authorization: renovateapi
```

Comment:
Would this not be 
```suggestion
  Authorization: Bearer renovateapi
```

Or

```suggestion
  Authorization: Token renovateapi
```

Or something similar? Some HTTP clients may expect there to be [a scheme in the authorization header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization)

Reply:
I _thought_ it should have such a prefix, but the internal testing docs we had indicated otherwise, so this is the primary reason I have paused for review

Reply:
no scheme is needed

Reply:
It would be nice if `Bearer` is optional as people may assume it's needed, for now I'll resolve and merge this